### DISCUSSION TITLE
Typescript Example of Extend Existing Components.

### DISCUSSION THREAD

**Discussion by JanderSilv** [uNmAnNeR/imaskjs#918] [Q&A]
I'm trying to use a custom input with React and the IMaskMixin component, but I didn't get it how to properly type the wrapper component. Any examples?

**Comment by uNmAnNeR** [comment]
here you can find an example of wrapping the native input
https://github.com/uNmAnNeR/imaskjs/blob/master/packages/react-imask/src/input.ts
